src/java.desktop/macosx/classes/sun/lwawt/macosx/CPlatformWindow.java

```diff
@@ -667,6 +667,44 @@ public void setVisible(boolean visible) {
             execute(CPlatformWindow::nativeSetNSWindowLocationByPlatform);
         }
 
+        // Manage the extended state when showing
+        if (visible) {
+                /* Frame or Dialog should be set property WINDOW_FULLSCREENABLE to true if the
+                Frame or Dialog is resizable.
+                **/
```

Comment:
Any reason for `**/` vs `*/`?

Reply:
This was in the original code so I didn't change it

Reply:
Understood, but maybe can still change it to match the rest of the file. The other comment blocks don't have this `**/` ending, so we can fix it while we make other edits to this.